browser/module.go

```diff
@@ -27,34 +26,34 @@ type (
 		Version  string
 	}
 
-	// ModuleInstance represents an instance of the JS module.
-	ModuleInstance struct {
+	// JSModuleInstance represents an instance of the JS module.
+	JSModuleInstance struct {
```

Comment:
I had to rename this to avoid the stuttering which was also causing a lint error.